.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+    - 'master'
+    tags:
+    - 'v*'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  ubuntu18:
+    name: ubuntu 18.04
+    runs-on: ubuntu-18.04
+    env:
+      CXX: g++-6
+      CC: gcc-6
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+      - name: install tools
+        run: sudo apt-get install -y g++-6 gcc-6 python3-pip python3-setuptools && pip3 install future pymavlink
```

Comment:
and without giving a specific gcc version...

Reply:
Again, that's what came from Travis, it was actually more work to keep it this way than to remove this constraint :-). I'll update that.

Reply:
I think this is leftover from when gcc 5.x was the default and we wanted at least 6... not sure. Thanks!